lib/ping_pong/util/ping_pong_table.js

```diff
@@ -11,6 +11,11 @@ const append = (pingPongTable, pingPrice, pongPrice) => {
 
 const extract = (pingPongTable, price) => {
   const pingItem = pingPongTable.find(item => item[0] === price)
```

Comment:
what are the odds of `pingPongTable` being empty? `item[0]` would break it too.

Reply:
```suggestion
  const pingItem = pingPongTable.find(item => item && item[0] === price)
```

Reply:
`pingPongTable` is a 2d array.. `pingPongTable` array being empty won't break the code .. and if there exists an item, it will always have the ping prices at index 0 and pong prices at index 1(handled this way by the `append` function).. so i think this check is unncessary